packages/augur-ui/src/modules/create-market/get-template.ts

```diff
@@ -395,14 +393,18 @@ export function getEventExpirationForExchange(
   if (!closing) return null;
   const dateYearSource = inputs.find(i => i.id === closing.inputDateYearId);
   const timeOffset = closing.userInputObject as TimeOffset;
-  if (dateYearSource && dateYearSource.setEndTime && timeOffset) {
-    const closingDateTime = getTemplateExchangeClosingWithBuffer(
-      dateYearSource.setEndTime,
+  if (dateYearSource && dateYearSource.userInput && timeOffset) {
```

Comment:
I believe `dateYearSource && dateYearSource.userInput` could be simplified to `dateYearSource?.userInput`